File: 0168-excel-sheet-column-title/0168-excel-sheet-column-title.py
class Solution:
    def convertToTitle(self, columnNumber: int) -> str:
        title = ""
        while 0 < columnNumber:
            rem = columnNumber % 26 or 26
            title = str(self.to_letter(rem)) + title
            columnNumber = (columnNumber - rem) // 26
        return title    

    def to_letter(self, x: int) -> str:
        return chr(x+64)

File: 0168-excel-sheet-column-title/0168-excel-sheet-column-title.go
func convertToTitle(columnNumber int) string {
    title := ""
    for 0 < columnNumber {
        rem := columnNumber % 26
        if rem == 0 {
            rem = 26
        }
        columnNumber = (columnNumber - rem) / 26
        title = toLetter(rem) + title
    }
    return title
}

func toLetter(x int) string {
    return string(rune(x+64))
}